lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
```

Comment:
i see what you mean. the tests could pick up local files like `~/.pryrc`, so yeah i think the tests would need FakeFS or something like it.